WiFiManager.cpp

```diff
@@ -78,6 +78,7 @@ WiFiManager::~WiFiManager()
     {
         DEBUG_WM(F("freeing allocated params!"));
         free(_params);
+        _params = NULL;
```

Comment:
Given that this is in the destructor the memory containing the pointer is being freed as well leaving no risk of a double free. This is not needed.

Reply:
I am going to merge it in case someone refactors code and moves this, it wont hurt.

Reply:
oh master, nm, ill just add it to dev